rero_ils/modules/loans/jsonschemas/loans/loan-ils-v0.0.1.json

```
@@ -125,6 +125,12 @@
         }
       }
     },
+    "to_anonymise": {
+      "title": "Set loan to be anonymised",
+      "description": "If enabled, loan will be anonymised. currently it is not returned for loan searches.",
+      "type": "boolean",
+      "default": false
+    },
```

```suggestion
    "to_anonymize": {
      "title": "Set loan to be anonymized",
      "description": "If enabled, loan will be anonymized. currently it is not returned for loan searches.",
      "type": "boolean",
      "default": false
    },
```
Actually, the description is a little bit misleading. "... set to be anonymized" is more accurate.